docs/pages/Theming/CustomTheme.md

```diff
@@ -1,7 +1,104 @@
 # Customized Themes
 
-## Creating a ThemeReference
+One of the powerful aspects of FURN theming is that it is heavily customizable. We recognize that users may have a need to change various parts of default themes, so we allow for several ways to specify overrides.
 
-## Theme reipces
+You can create a custom theme by either building on top of an existing ThemeReference or building one from scratch. This then can be used by [passing it into our ThemeProvider](./Basics.md), which makes the theme available for all child components of the provider.
 
-## Adding theme recipes to THemeReference
+## Different aspects of customization
+
+There's two main ways to customize a FURN theme - you can specify how to customize components, or you can customize theme tokens directly by overriding certain tokens.
+
+### Components
+
+The FURN theme has a components property which can be used to customize components:
+
+```ts
+interface Theme {
+  ...
+  components: {
+    [key: string]: object,
+  };
+  ...
+}
+```
+
+<font size=1>(Taken from the [Theme type definition](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theme-types/src/Theme.types.ts).)</font>
+
+In order to have the theme override a particular component's styling, you'll need to specify a component's name and provide some overrides for the component's settings. The settings are an object that specifies how a component's tokens or slot's style are filled out.
+
+```ts
+components: {
+  <ComponentName>: {
+    <tokenName>: <overrideValue>
+  },
+},
+```
+
+For example, if you want to change the settings on the [Button component](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.tsx), you could do something like the following:
+
+```ts
+components: {
+  Button: {
+    backgroundColor: 'primaryButtonBackground', // Different semantic color
+    color: 'neutralBackground1', // Alias token
+    hovered: {
+      backgroundColor: theme.host.colors['AppShade10'], // Get brand color from host
+      color: theme.host.colors['Gray96'], // Get gray from host
+      borderColor: theme.host.palette.Bkg, // Get entry from palette from host
+    },
+  },
+},
+```
```

Comment:
nit: These overrides are for the experimental button, but we link the original button here. Also, could we just link button.settings/button.styling?

Reply:
Ah does this only work for the newer compose framework?

Reply:
The structure of the object you passed in is specific to the experimental buttons' settings, specifically the nested hover. It works with the old framework, the shape would just look different since you have stuff like `_precedences` in there which aren't in the new framework.

Reply:
Ah okay my bad. I copied the structure from something else and changed the component without really thinking about it. I'll fix it up.

Reply:
Should be fixed now, thanks for catching!